site/data/charts/docker.json

```diff
@@ -50,6 +50,7 @@
     ["2021-03-01", 9032, 4747115, 1174371, 3163, ""],
     ["2021-04-01", 7203, 968932, 207199, 3147, ""],
     ["2021-05-01", 1733, 11146247, 636749, 2377, ""],
-    ["2021-06-01", 1467, 10890089, 7211522, 2222, ""]
+    ["2021-06-01", 1467, 10890089, 7211522, 2222, ""],
+    ["2021-07-01", 98206, 8350573, 7133123, 1911, ""]
```

Comment:
Umm that’s quite a change

Reply:
yeah, wish I knew what was going on here :P